### PR TITLE
refactor : update component navigation menuand make it responsive

### PR DESCRIPTION
**Pull request by viru2001**
## change logs

1. updated styling of component navigation
2. added hamburger navigation for smaller screens

updated comp navigation

hamburger comp nav

### CODE REVIEW

File: styles/docs.css [hunk 1]
```diff
@@ -5,31 +5,108 @@
...
+#btn-hamburger {
+  display: none;
+}
+
+#nav-menu-hamburger {
+  border-bottom: solid 2px var(--secondary-dark-color);
+}
+
+.component-nav-hamburg {
+  height: 45vh;
```
Comment (srv-pranjal) [inline styles/docs.css:51]: If there isn't any specific reason to set the height to 45vh, you should probably increase it to 100%.

**Review by srv-pranjal — COMMENTED** [review]
Good work. One suggestion is that you can try to close the responsive nav after clicking on any component as it remains there even after getting scrolled to the desired component which blocks the UI.

**Comment by viru2001** [comment]
> Good work. One suggestion is that you can try to close the responsive nav after clicking on any component as it remains there even after getting scrolled to the desired component which blocks the UI.

If we click on the hamburger icon again then it will close. 
I will try to close the responsive nav after clicking on any component.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Pull request merged**